### ISSUE TITLE
Parse error when listing incidents

### ISSUE THREAD

**Issue by huskercane** [technicalpickles/pager_duty-connection#16]
I get following, most probably because there are no incidents pending

```
2.3.0 :010 >   response = pagerduty.get('incidents')
RuntimeError: Can't parse times of NilClass:
	from /home/ibmadmin/.rvm/gems/ruby-2.3.0@global/gems/pager_duty-connection-0.2.0/lib/pager_duty/connection.rb:108:in `parse'
	from /home/ibmadmin/.rvm/gems/ruby-2.3.0@global/gems/faraday-0.9.2/lib/faraday/response.rb:16:in `on_complete'
	from /home/ibmadmin/.rvm/gems/ruby-2.3.0@global/gems/faraday-0.9.2/lib/faraday/response.rb:9:in `block in call'
	from /home/ibmadmin/.rvm/gems/ruby-2.3.0@global/gems/faraday-0.9.2/lib/faraday/response.rb:57:in `on_complete'
	from /home/ibmadmin/.rvm/gems/ruby-2.3.0@global/gems/faraday-0.9.2/lib/faraday/response.rb:8:in `call'
	from /home/ibmadmin/.rvm/gems/ruby-2.3.0@global/gems/faraday_middleware-0.9.2/lib/faraday_middleware/request/encode_json.rb:23:in `call'
	from /home/ibmadmin/.rvm/gems/ruby-2.3.0@global/gems/pager_duty-connection-0.2.0/lib/pager_duty/connection.rb:59:in `call'
	from /home/ibmadmin/.rvm/gems/ruby-2.3.0@global/gems/pager_duty-connection-0.2.0/lib/pager_duty/connection.rb:21:in `call'
	from /home/ibmadmin/.rvm/gems/ruby-2.3.0@global/gems/pager_duty-connection-0.2.0/lib/pager_duty/connection.rb:32:in `call'
```

**Comment by technicalpickles** [comment]
> most probably because there are no incidents pending

What makes you think that? I've used this when there haven't been any incidents.

Looking at the first line in question https://github.com/technicalpickles/pager_duty-connection/blob/master/lib/pager_duty/connection.rb#L108 , it looks like it's checking the `body` of the response, and that that is nil. It's possible that PagerDuty is returning empty string, which gets decoded from JSON as nil, but I don't think it's likely.

I would check your API tokens, but I think that would surface itself a different way.

PS: I edited your request to format the output
